rskj-core/src/test/java/co/rsk/peg/BridgeStorageProviderTest.java

```diff
@@ -2925,26 +2904,26 @@ public void getFlyoverFederationInformation_beforeRSKIP176_ok() {
 
         Keccak256 derivationHash = PegTestUtils.createHash3(2);
         byte[] federationRedeemScriptHash = new byte[]{(byte) 0xaa};
-        byte[] flyoverFederationRedeemScriptHash = new byte[]{(byte)0x22};
+        byte[] flyoverFederationRedeemScriptHash = new byte[]{(byte) 0x22};
         FlyoverFederationInformation flyoverFederationInformation = new FlyoverFederationInformation(
-            derivationHash,
-            federationRedeemScriptHash,
-            flyoverFederationRedeemScriptHash
+                derivationHash,
+                federationRedeemScriptHash,
+                flyoverFederationRedeemScriptHash
         );
 
-        when(repository.getStorageBytes(
-            PrecompiledContracts.BRIDGE_ADDR,
-            DataWord.fromLongString("fastBridgeFederationInformation-" + Hex.toHexString(flyoverFederationRedeemScriptHash)))
+        lenient().when(repository.getStorageBytes(
```

Comment:
@josedahlquist I had to add this `lenient()` after rebasing with master (containing Java and Mockito upgrade changes), otherwise it was complaining about _Unnecessary stubbings detected_ when using newer Mockito for this mock.

Could your team please double check if the test is doing what it is expected? Then we can decide what to do with those lines.

Thanks!

Reply:
@josedahlquist , @marcos-iov 
this one could be similar to the one discussed [here](https://github.com/rsksmart/rskj/pull/1812#discussion_r957899887), maybe you can take a look as well when merged.